proto/ibc/channel/query.proto

```diff
@@ -0,0 +1,183 @@
+syntax = "proto3";
+package ibc.channel;
+
+import "gogoproto/gogo.proto";
+import "cosmos/query/pagination.proto";
+import "ibc/channel/channel.proto";
+
+option go_package = "github.com/cosmos/cosmos-sdk/x/ibc/04-channel/types";
+
+// Query provides defines the gRPC querier service
+service Query {
```

Comment:
im not sure the path but don't all these need to be annotated to support grpc-gateway?

Reply:
ignore, I guess not.